pkg/controller/machine/queue.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package machine
+
+import (
+	"time"
+
+	"github.com/golang/glog"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/kubernetes-incubator/apiserver-builder/pkg/controller"
+	"sigs.k8s.io/cluster-api/pkg/controller/config"
+)
+
+func (c *MachineController) RunAsync(stopCh <-chan struct{}) {
+	for _, q := range c.Informers.WorkerQueues {
+		c.StartWorkerQueue(q, stopCh)
+	}
+	controller.GetDefaults(c.controller).Run(stopCh)
```

Comment:
This does nothing. It calls https://github.com/kubernetes-sigs/cluster-api/blob/df991ef535792b2bbec5242d63316cfd153ab5e6/vendor/github.com/kubernetes-incubator/apiserver-builder/pkg/controller/controller.go#L131-L137 
which checks for 
https://github.com/kubernetes-sigs/cluster-api/blob/df991ef535792b2bbec5242d63316cfd153ab5e6/vendor/github.com/kubernetes-incubator/apiserver-builder/pkg/controller/controller.go#L139-L141
`Run(stopCh <-chan struct{})` function on our `c.controller` struct. We don't have such function, so it fallbacks to dummy default function.

Reply:
I understood that. My original intention was to keep consistency with api-server builder. Remove that.